File: OPENAPI-REST-API/openapi-client/scala-gatling/src/gatling/scala/org/openapitools/client/model/VpsTemplateRow.scala

package org.openapitools.client.model


case class VpsTemplateRow (
    _template_id: Option[String],
    _template_type: Option[String],
    _template_os: Option[String],
    _template_version: Option[String],
    _template_bits: Option[String],
    _template_file: Option[String],
    _template_available: Option[String],
    _template_name: Option[String],
    _template_dir: Option[String]
)
object VpsTemplateRow {
    def toStringBody(var_template_id: Object, var_template_type: Object, var_template_os: Object, var_template_version: Object, var_template_bits: Object, var_template_file: Object, var_template_available: Object, var_template_name: Object, var_template_dir: Object) =
        s"""
        | {
        | "template_id":$var_template_id,"template_type":$var_template_type,"template_os":$var_template_os,"template_version":$var_template_version,"template_bits":$var_template_bits,"template_file":$var_template_file,"template_available":$var_template_available,"template_name":$var_template_name,"template_dir":$var_template_dir
        | }
        """.stripMargin
}

File: OPENAPI-REST-API/openapi-client/zapier/models/VpsTemplateRow.js
const utils = require('../utils/utils');

module.exports = {
    fields: (prefix = '', isInput = true, isArrayChild = false) => {
        const {keyPrefix, labelPrefix} = utils.buildKeyAndLabel(prefix, isInput, isArrayChild)
        return [
            {
                key: `${keyPrefix}template_id`,
                label: `[${labelPrefix}template_id]`,
                type: 'string',
            },
            {
                key: `${keyPrefix}template_type`,
                label: `[${labelPrefix}template_type]`,
                type: 'string',
            },
            {
                key: `${keyPrefix}template_os`,
                label: `[${labelPrefix}template_os]`,
                type: 'string',
            },
            {
                key: `${keyPrefix}template_version`,
                label: `[${labelPrefix}template_version]`,
                type: 'string',
            },
            {
                key: `${keyPrefix}template_bits`,
                label: `[${labelPrefix}template_bits]`,
                type: 'string',
            },
            {
                key: `${keyPrefix}template_file`,
                label: `[${labelPrefix}template_file]`,
                type: 'string',
            },
            {
                key: `${keyPrefix}template_available`,
                label: `[${labelPrefix}template_available]`,
                type: 'string',
            },
            {
                key: `${keyPrefix}template_name`,
                label: `[${labelPrefix}template_name]`,
                type: 'string',
            },
            {
                key: `${keyPrefix}template_dir`,
                label: `[${labelPrefix}template_dir]`,
                type: 'string',
            },
        ]
    },
    mapping: (bundle, prefix = '') => {
        const {keyPrefix} = utils.buildKeyAndLabel(prefix)
        return {
            'template_id': bundle.inputData?.[`${keyPrefix}template_id`],
            'template_type': bundle.inputData?.[`${keyPrefix}template_type`],
            'template_os': bundle.inputData?.[`${keyPrefix}template_os`],
            'template_version': bundle.inputData?.[`${keyPrefix}template_version`],
            'template_bits': bundle.inputData?.[`${keyPrefix}template_bits`],
            'template_file': bundle.inputData?.[`${keyPrefix}template_file`],
            'template_available': bundle.inputData?.[`${keyPrefix}template_available`],
            'template_name': bundle.inputData?.[`${keyPrefix}template_name`],
            'template_dir': bundle.inputData?.[`${keyPrefix}template_dir`],
        }
    },
}
